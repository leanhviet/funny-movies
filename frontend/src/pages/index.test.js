// Libs
import React from 'react'
import { shallow } from 'enzyme'

// Components
import { App } from './index'

describe('App', () => {
  const wrapper = shallow(<App />)

  it('should be defined App', () => {
    expect(wrapper).toBeDefined()
  })

  // Testing Snapshot
  it('should be render snapshot App', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
