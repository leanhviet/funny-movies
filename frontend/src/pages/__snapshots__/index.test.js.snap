// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should be render snapshot App 1`] = `
<BrowserRouter>
  <AppRouter />
</BrowserRouter>
`;
